stdlib/Logging/src/ConsoleLogger.jl

```diff
@@ -148,7 +148,9 @@ function handle_message(logger::ConsoleLogger, level::LogLevel, message, _module
     # Format lines as text with appropriate indentation and with a box
     # decoration on the left.
     color, prefix, suffix = logger.meta_formatter(level, _module, group, id, filepath, line)::Tuple{Union{Symbol,Int},String,String}
-    color = StyledStrings.Face(foreground=StyledStrings.Legacy.legacy_color(color))
+    if (lcolor = StyledStrings.Legacy.legacy_color(color)) |> !isnothing
```

Comment:
style nit: assignment in an `if` statement is not typically permitted style in Base, nor is `|>`
```suggestion
    lcolor = StyledStrings.Legacy.legacy_color(color)
    if !isnothing(lcolor)
```

Reply:
Ah ok, good to know (aside: is the code style that `Base` uses documented anywhere?).

Reply:
There sort of is (https://docs.julialang.org/en/v1/manual/style-guide/), though neither of those patterns is mentioned in it. They don't generally come up often, which is sort of why they both don't get mentioned there and aren't generally used as the style in Base, since derives mostly just from what people commonly do